### DISCUSSION TITLE
Offline switch

### DISCUSSION THREAD

**Discussion by pelupessy** [Heroic-Games-Launcher/HeroicGamesLauncher#4141] [Ideas]
Problem Description:
At the moment, those who wish to stop heroic (and components) from going online must disable their internet connection.

Solution: a conveniently located "offline mode" switch.

**Comment by arrowgent** [comment]
the only way to do that is run heroic and the games in a sandbox with the network feature off

im going to be testing **firejail** to see how much it interferes with wine/proton and the heroic sub modules: gogdl, etc
and how much overhead this causes if it effects gameplay from within the sandbox

**Comment by arrowgent** [comment]
`firejail --noprofile --net=none "$@"`
the variable $@ would be the heroic launcher
or eg: `fjnet heroic`

blocks all heroic internet access and game access
